specs/APIReview_TextureStream.md

```diff
@@ -0,0 +1,577 @@
+  /// corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2StagingTextureStream* sender,
+      [in] IUnknown* args);
+}
+[uuid(431721e0-0f18-4d7b-bd4d-e5b1522bb110), object, pointer_default(unique)]
+interface ICoreWebView2StagingRenderAdapterLUIDChangedEventHandler : IUnknown {
+  /// Called to provide the implementer with the event args for the
+  /// corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2StagingEnvironment * sender,
+      [in] IUnknown* args);
+}
+/// This is the callback for web texture.
+[uuid(9ea4228c-295a-11ed-a261-0242ac120002), object, pointer_default(unique)]
+interface ICoreWebView2StagingTextureStreamWebTextureReceivedEventHandler : IUnknown {
+  /// Called to provide the implementer with the event args for the
+  /// corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2StagingTextureStream* sender,
+      [in] ICoreWebView2StagingTextureStreamWebTextureReceivedEventArgs* args);
+}
+
+/// This is the event args interface for web texture.
+[uuid(a4c2fa3a-295a-11ed-a261-0242ac120002), object, pointer_default(unique)]
+interface ICoreWebView2StagingTextureStreamWebTextureReceivedEventArgs : IUnknown {
+  /// Return ICoreWebView2StagingWebTexture object.
+  /// The call does not create new ICoreWebView2StagingWebTexture object, instead
+  /// returns the same object.
+
+  /// The TextureBuffer handle will be reused when ICoreWebView2StagingWebTexture
+  /// object is released. So, the host should not refer handle or resource of
+  /// the ICoreWebView2StagingWebTexture after its release.
+
+  HRESULT GetWebTexture([out, retval] ICoreWebView2StagingWebTexture** value);
+}
+
+/// This is the callback for web texture stop.
+[uuid(77eb4638-2f05-11ed-a261-0242ac120002), object, pointer_default(unique)]
+interface ICoreWebView2StagingTextureStreamWebTextureStreamStoppedEventHandler : IUnknown {
+  /// Called to provide the implementer with the event args for the
+  /// corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2StagingTextureStream* sender,
+      [in] IUnknown* args);
+}
+
+/// Received TextureBuffer that the renderer writes to so that the host
+/// will read on it.
+[uuid(b94265ae-4c1e-11ed-bdc3-0242ac120002), object, pointer_default(unique)]
+interface ICoreWebView2StagingWebTexture : IUnknown {
+  /// TextureBuffer handle. The handle's lifetime is owned by the
+  /// ICoreWebView2StagingTextureStream object so the host must not close it.
+  /// The same handle value will be used for same TextureBuffer so the host can use
+  /// handle value as a unique TextureBuffer key.
+  /// If the host opens its own resources by handle, then it is suggested
+  /// that the host removes those resources when the handle's texture size
+  /// is changed because the browser also removed previously allocated different
+  /// sized buffers when image size is changed.
+  [propget] HRESULT Handle([out, retval] HANDLE* handle);

```

Comment:
Just make sure to update these docs the same as changes to the other Texture interface's docs.

Reply:
use value for property.